src/lib/components/TruncateDescription.js

```diff
@@ -0,0 +1,40 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2020 Graz University of Technology.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import PropTypes from 'prop-types';
+import _truncate from 'lodash/truncate';
+import React, { Component } from 'react';
+
+export class TruncateDescription extends Component {
+    state = {
+        texts: '',
+    };
+    constructor(props) {
+        super(props);
+        this.state = {
+            texts: _truncate(this.props.texts, { length: this.props.length }),
+        };
+    }
+
+    render() {
+        return (
+            <span>{this.state.texts}
```

Comment:
I think for now I would just take these 2 lines and truncate the text directly [here](https://github.com/inveniosoftware/react-invenio-deposit/pull/204/files#diff-46acefff300d8dc2d2bcd541320b7f007e567a51999e24ee5b8c622d997476cbR99). If we reuse the same way in other places, then we can extract the logic to a reusable component :)